File: CS/WinSample.Module.Win/SupportCheckItemsController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;
using DevExpress.ExpressApp.Win.Templates;
using DevExpress.XtraBars;
using DevExpress.ExpressApp.Win.Templates.ActionContainers.Items;

namespace WinSample.Module.Win {
    public partial class SupportCheckItemsController : WindowController {
        protected override void OnActivated() {
            base.OnActivated();
            DefaultBarActionItemsFactory.CustomizeActionControl += new EventHandler<DevExpress.ExpressApp.Win.Templates.ActionContainers.CustomizeActionControlEventArgs<ActionBase>>(DefaultBarActionItemsFactory_CustomizeActionControl);
        }
        void DefaultBarActionItemsFactory_CustomizeActionControl(object sender, DevExpress.ExpressApp.Win.Templates.ActionContainers.CustomizeActionControlEventArgs<ActionBase> e) {
            if (e.Action is CheckableSimpleAction) {
                e.ActionControl = new CheckActionItem(e.Action, e.ActionControl.Control.Manager);
            }
        }
    }
}

File: VB/WinSample.Module.Win/SupportCheckItemsController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base
Imports DevExpress.ExpressApp.Win.Templates
Imports DevExpress.XtraBars
Imports DevExpress.ExpressApp.Win.Templates.ActionContainers.Items

Namespace WinSample.Module.Win
	Partial Public Class SupportCheckItemsController
		Inherits WindowController
		Protected Overrides Sub OnActivated()
			MyBase.OnActivated()
			AddHandler DefaultBarActionItemsFactory.CustomizeActionControl, AddressOf DefaultBarActionItemsFactory_CustomizeActionControl
		End Sub
		Private Sub DefaultBarActionItemsFactory_CustomizeActionControl(ByVal sender As Object, ByVal e As DevExpress.ExpressApp.Win.Templates.ActionContainers.CustomizeActionControlEventArgs(Of ActionBase))
			If TypeOf e.Action Is CheckableSimpleAction Then
				e.ActionControl = New CheckActionItem(e.Action, e.ActionControl.Control.Manager)
			End If
		End Sub
	End Class
End Namespace
